tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -479,6 +458,9 @@ let x =
         script.Eval(code) |> ignoreValue
         Assert.False(foundInner)
 
+// Fails in NETFRAMEWORK with exception
```

Comment:
Does it mean it wasn't failing before?

Reply:
Ah, no, iirc this whole test project was excluded on net472 and I re-enabled it for some reason.